src/components/AttentionBox/AttentionBox.scss

```diff
@@ -13,6 +13,8 @@
 
   &__title {
     margin: 0;
+    font: var(--font-h5);
```

Comment:
why? Attention box using heading for heading component so this override is weird. changing the heading type was not enough?

Reply:
After check with yael: we need to fix the weight of h5 heading to 700. Regarding the font it should work without override, what is the problem?

Reply:
no need in that - agree